src/mbgl/map/tile_worker.cpp

```diff
@@ -41,14 +41,13 @@ TileParseResult TileWorker::parseAllLayers(std::vector<std::unique_ptr<StyleLaye
                                            PlacementConfig config) {
     // We're doing a fresh parse of the tile, because the underlying data has changed.
     pending.clear();
+    placementPending.clear();
     partialParse = false;
+    placementConfig = config;
```

Comment:
Instead of storing a copy of `placementConfig`, would it be better to pass the correct value to `TileWorker::parsePendingLayers`, and then pass it through as a parameter to `placeLayers` everywhere?